src/main/java/seedu/duke/timetable/Timetable.java

```diff
@@ -0,0 +1,206 @@
+package seedu.duke.timetable;
+
+import seedu.duke.exceptions.InvalidLessonDayException;
+import seedu.duke.exceptions.InvalidTimeFormatException;
+import seedu.duke.exceptions.InvalidTimingException;
+import seedu.duke.exceptions.LessonNotFoundException;
+import seedu.duke.exceptions.TimetableClashException;
+import seedu.duke.ui.Ui;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Timetable {
+    private HashMap<String, ArrayList<Lesson>> userTimetable;
+
+    public Timetable() {
+        userTimetable = new HashMap<String, ArrayList<Lesson>>();
+        ArrayList<Lesson> mondayList = new ArrayList<Lesson>();
+        userTimetable.put("monday", mondayList);
```

Comment:
The value of days could be implemented as en enumeration instead. It could help with reusability and case handling for the rest of the program, also makes it easier to reason about the code.